FreeRTOS/FreeRTOS-Kernel
# Fixed some bugs in documentation and made it look consistent

Fixed some documentation bugs. 

Description
-----------
There have been some double-opened `<pre>` tags resulting in parts of documentation not being rendered. Many patch lines are only clean code, but some of them are real documentation bugs. 

Test Steps
-----------
One example is for xMessageBufferSend that had no signature in doxygen generated documentation:
https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/4b353bfd7aa009d553f31b4e47dfb49a9f3c75b1/include/message_buffer.h#L213-L218

Related Issue
-----------
None


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
